dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -156,6 +156,14 @@ JSON_FIELDS_INVALID_CHARACHTER=TRAS3011W: An invalid character was provided in t
 JSON_FIELDS_INVALID_CHARACHTER.explanation= Only letters, numbers, and : are accepted in the JSON logging field name mapping property
 JSON_FIELDS_INVALID_CHARACHTER.useraction= Change your JSON logging field name mapping property value to remove invalid characters
 
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING=TRAS3012W: The log rollover start time property is incorrectly formatted. The value will be set to a default of '00:00', representing midnight.
```

Comment:
suggested edit:
The value will be set to a default of '00:00', representing midnight.
--->
As a result, the value is set to a default of '00:00', which represents midnight.